test/test_linop.jl

```diff
@@ -100,6 +100,36 @@ function test_linop()
           @test norm(op' * v - A' * v) <= rtol * norm(A)
         end
       end
+
+      v = simple_vector(Float64, nrow)
+
+      A = simple_matrix(ComplexF64, nrow, nrow)
+      A = A + A'; A = A + transpose(A) # Symmetric and Hermitian
```

Comment:
Why the two operations here?

Reply:
So it's symmetric and Hermitian

Reply:
What does that even mean "symmetric and Hermitian"? Only real symmetric matrices are "symmetric and Hermitian", no?

Reply:
Yes, but the `eltype(A)` is `ComplexF64` in this case, and we have a case for that [here](https://github.com/JuliaSmoothOptimizers/LinearOperators.jl/pull/118/files#diff-4fa936477f81d0e7cf6e57db36419ac7R89) so I'm making sure it's covered.

Reply:
Gotcha. Would you mind adding a comment about that in the test? I found it confusing.

Reply:
Done.